packages/turf/test.ts

```diff
@@ -331,7 +331,7 @@ const turfTypescriptPath = path.join(__dirname, "..", "turf-*", "index.d.ts");
 
 // Test Strings
 const requireString = `import test from 'tape';
-import * as turf from './dist/esm/index.js';
+import * as turf from './index.ts';
```

Comment:
Jumping into the built output gets us away from the tsconfig paths settings, so we have to hit the index.ts instead.